sdks/go/pkg/beam/core/runtime/graphx/translate.go

```diff
@@ -0,0 +1,651 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package graphx
+
+import (
+	"fmt"
+
+	"strconv"
+	"strings"
+
+	"github.com/apache/beam/sdks/go/pkg/beam/core/graph"
+	"github.com/apache/beam/sdks/go/pkg/beam/core/graph/window"
+	"github.com/apache/beam/sdks/go/pkg/beam/core/runtime/graphx/v1"
+	"github.com/apache/beam/sdks/go/pkg/beam/core/util/protox"
+	fnpb "github.com/apache/beam/sdks/go/pkg/beam/model/fnexecution_v1"
+	pb "github.com/apache/beam/sdks/go/pkg/beam/model/pipeline_v1"
+	"github.com/golang/protobuf/proto"
+)
+
+const (
+	// Model constants
+
+	urnImpulse = "urn:beam:transform:impulse:v1" // to appear?
```

Comment:
<!--new_thread; commit:c5876af98386a806693a64b6dc5905f2397c35fb; resolved:0-->
rm " // to appear?" It doesn't add anything meaningful to the reader, let alone suggest anything that would need to change if it has or has not been implemented in the model.

Instead link/reference to the appropriate Jira proposal if it's only that far in.
Otherwise it's not worth calling attention to it.